### DISCUSSION TITLE
weather widget

### DISCUSSION THREAD

**Discussion by sauliusputinas** [openbookings26/ContaCam#17] [Ideas]
Hi 
I came to idea of new functionality which could be added to contacam.
I suspect most of users use your soft to monitor some remote places - so it is interesting to see not only  video, but also know about weather conditions in remote place.
That is why I thought maybe there would be possibility to integrate some widget which is showing remote location's temperature like it is done in : https://balticlivecam.com/cameras/latvia/jurmala/majori-beach/
![unnamed](https://github.com/frakman1/ContaCam/assets/49292639/e2bfe447-c1d2-45f2-94b5-1077c3be8184)

So what I did - I googled some weather widgets and one most suitable for me looked that one :
https://weatherwidget.io/
I generated code and placed it into the top of your index_rootdir.php

![unnamed-1](https://github.com/frakman1/ContaCam/assets/49292639/21aa2910-d984-484b-ae03-5bee22e727d6)

And here how it looks the first page after logging in - I achieved wanted result ;)

![278333182-7de7fa51-2cd9-4fa5-90bf-67e86d3f4b56](https://github.com/frakman1/ContaCam/assets/49292639/90482f4f-36f5-470d-83bc-6eede45ae794)

So maybe there could be some possibility to add similar functionality to application administration window for other users
Just an idea ;)

**Comment by openbookings26** [comment]
**Thank you** for sharing this idea. I love being able to add a weather widget to the Camera dashboard!
I doubt this sort of feature will make it into ContaCam but I love the ease with which it can be manually added.

I tried it myself with a different [site](https://www.weatherwx.com/forecast.php). After putting in your location, it gives you several options to pick from. I like this one because it has more details and a link to an Hourly forecast and a Radar map for more information.

![image](https://github.com/frakman1/ContaCam/assets/5826484/1b37f0b3-cc25-4a58-929c-f88c52484e96)

**Comment by sauliusputinas** [comment]
And after contacam version update installation you have to make these manipulations once gain :)
As well as others done in summarysnapshot.php to change the grid size

**Comment by MisterWho** [comment]
Thank you so much. Great idea! I'll be looking over here for more customization ideas. Kudos!

![imagem](https://github.com/frakman1/ContaCam/assets/1645474/4b49bb65-d50d-48fd-9b38-5d0c8f7d95dc)
